src/mediatype-icon.ts

```diff
@@ -32,7 +32,7 @@ export class MediatypeIcon extends LitElement {
   }
 
   render() {
-    if (!this.mediatype) {
+    if (!this.mediatype || !this.displayMediatype) {
```

Comment:
You don't need the `!this.mediatype` check anymore since you're not referencing it in the `render()` method.